tests/Fixtures/inc/Engine/CDN/RocketCDN/NoticesSubscriber/displayRocketcdnCta.php

```diff
@@ -25,7 +25,7 @@
 		<p class="wpr-rocketcdn-cta-subtitle">Speed up your website thanks to:</p>
 		<div class="wpr-flex">
 			<ul class="wpr-rocketcdn-features">
-				<li class="wpr-rocketcdn-feature wpr-rocketcdn-bandwidth">High performance Content Delivery Network (CDN) with <strong>unlimited bandwith</strong></li>
+				<li class="wpr-rocketcdn-feature wpr-rocketcdn-bandwidth">High performance Content Delivery Network (CDN) with <strong>unlimited bandwidth</strong></li>
```

Comment:
Why those strings are not translatable?
@mostafa-hisham Do u think it's the time to change it to use the WordPress translation functions?

Reply:
those are tests, sorry I didn't notice that :D